functions/src/backfillToTypesenseFromFirestore.js

```diff
@@ -40,8 +40,16 @@ module.exports = functions.handler.firestore.document
         return false;
       }
 
-      const querySnapshot =
-        await admin.firestore().collection(config.firestoreCollectionPath).get();
+      const isSubcollection = config.firestoreCollectionPath.includes("/");
+      let query;
+      if (isSubcollection) {
+        const collectionGroup = config.firestoreCollectionPath.split("/").pop();
+        query = admin.firestore().collectionGroup(collectionGroup);
```

Comment:
Does this support collections with dynamic path names? Eg: `/users/{user_id}/companies`

Reply:
Yes it will work like this